### PR TITLE
Move CLI scripts into a single command

### PR DESCRIPTION
**Pull request by twe4ked**

### CODE REVIEW

File: scripts/request [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  c.example 'Amend the title of a Todo', %Q{#{script_name} add -i 0b341422-c516-4ee4-8f3e-ef1992dfff32 -t "My task"}
+  c.option '-i ID', '--id ID', 'Existing Todo ID'
+  c.option '-t TITLE', '--title TITLE', 'Title'
+  c.option '-d DESCRIPTION', '--description DESCRIPTION', 'Description'
+  c.option '-D DUE_DATE', '--due_date DUE_DATE', 'Due date'
+  c.option '-s STAKEHOLDER_EMAIL', '--stakeholder_email STAKEHOLDER_EMAIL', 'Stakeholder email'
+  c.action do |args, options|
+    todo_id = options.id
+    unless todo_id
+      $stderr.puts "Error: you must specify a todo_id to amend"
```
Comment (andrewgr) [inline scripts/request]: `todo_id` is called "Todo ID" in line 91. Maybe this message should say something like "Error: Todo ID is not specified"?

File: scripts/request [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      exit 1
+    end
+    payload = slice_and_compact(options.default, :title, :description, :due_date, :stakeholder_email)
+    puts "Amending todo [#{todo_id}]: #{payload}"
+    Request.call(:put, "http://localhost:3000/todo/#{todo_id}", payload)
```
Comment (andrewgr) [inline scripts/request]: Is it always 3000?

Comment (grassdog) [inline scripts/request]: If running from foreman it should be.

Comment (twe4ked) [inline scripts/request]: `Procfile` specifies 3000.

Comment (andrewgr) [inline scripts/request]: Yep. Thanks.

**Comment by twe4ked** [comment]
Thanks for the reviews. I've pushed some changes @grassdog @andrewgr

**Pull request merged**